scripts/cnode-helper-scripts/prereqs.sh

```diff
@@ -153,6 +153,7 @@ if [[ "${INTERACTIVE}" = 'Y' ]]; then
   clear
   CNODE_PATH=$(get_input "Please enter the project path" ${CNODE_PATH})
   CNODE_NAME=$(get_input "Please enter directory name" ${CNODE_NAME})
+  CNODE_NAME=${CNODE_NAME//[^[:alnum:]]/_}
```

Comment:
I think we cannot have this here without updating the update mechanism in other scripts - as the tools doing update in place would likely place wrong vnames

Reply:
ny bad, we're not allowing different foldername from env var, all good